pkg/virt-handler/vm_test.go

```diff
@@ -988,6 +988,123 @@ var _ = Describe("VirtualMachineInstance", func() {
 			))
 		})
 
+		It("should ignore qemu paused on migration", func() {
```

Comment:
Can you dedup these cases into a table?

Reply:
Like all of the ones unders Describe ? or a subnet related to paused ?

Reply:
The two you've added. They're pretty similar, no?